articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+# How to build a static site with GatsbyJs
+![Hero image](/engineering-education/howto-build-astatic-site-with-gatsbyjs/heroimage.png)
+[*Image Source: Unsplash](https://unsplash.com/photos/uyfohHiTxho)
+
+When I first heard about GatsbyJS, I was confused about what it was, and how I would get started. As time went on, I learned more about GatsbyJS and found out it is awesome. My goal for this article is not to confuse you about GatsbyJS, but to help you get started with building your very first static site using GatsbyJS.
+
+GatsbyJS is used to build websites powered by [JAMStack](https://jamstack.org/what-is-jamstack/). JAMStack is a modern framework for creating websites and software. The “JAM” is an acronym for JavaScript, APIs, and HTML markup, JAMStack sites do not need a database, unlike websites designed using WordPress or Drupal. 
```

Comment:
- "The JAMstack is..."
- It is common to refer to is as "The JAMstack" as opposed to JAMstack alone - except when asking things like "What is JAMstack?"
- Run-on sentence between markup and JAMstack

Reply:
Done